bindings/php/README.md

```diff
@@ -1,32 +1,82 @@
-# OpenDAL PHP Binding (WIP)
+# OpenDAL PHP Binding
+
+## Example
+
+```php
+use OpenDAL\Operator;
+
+$op = new Operator("fs", ["root" => "/tmp"]);
+$op->write("test.txt", "hello world");
+
+echo $op->read("test.txt"); // hello world
+```
 
 ## Requirements
 
 * PHP 8.1+
+* Composer
 
-## Development
+## Install Extension
```

Comment:
Is there a way to distribute native exetension so that users only need to add fields in `composer` instead? I expect users don't need to know about rust core details and just use `opendal` as a native php package.

Reply:
The best option for us seems submit opendal to PECL.

Reply:
ok, pecl just provide source and users still need to build and compile opendal.